File: ServerCode/ServerCode/ServerCode/Model/Entity/Post.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ServerCode.Model.Entity
{
    public class Post
    {
        public int Id { get; set; }
        public string PostText { get; set; }
        public DateTime DateSendMessage { get; set; }
        public string OwnerId { get; set; }
        public User Owner { get; set; }
        public int ForumId { get; set; }
        public Forum Forum { get; set; }
    }
}

File: ClientApp/ClientApp/src/app/models/Forum/post.ts
import { UserProfile } from '../Chats/userProfile';
import { Forum } from './forum';

export interface Post {
    id?: number;
    postText?: string;
    dateSendMessage?: Date;
    ownerId?: string;
    owner?: UserProfile;
    forumId?: number;
    forum?: Forum;
    reciver?;
}
